### ISSUE TITLE
CloudCompare .ply file import errors

### ISSUE THREAD

**Issue by unitycoder** [unitycoder/UnityPointCloudViewer#34]
.ply file exported from CloudCompare causes errors (because header is different)

cloudcompare header
```
ply
format ascii 1.0
comment Author: CloudCompare (TELECOM PARISTECH/EDF R&D)
obj_info Generated by CloudCompare!
element vertex 6137
property float x
property float y
property float z
property uchar red
property uchar green
property uchar blue
end_header
```

typical header
```
ply
format ascii 1.0
comment VCGLIB generated
element vertex 2155617
property float x
property float y
property float z
property uchar red
property uchar green
property uchar blue
property uchar alpha
element face 4305818
property list uchar int vertex_indices
end_header
```

**Comment by unitycoder** [comment]
fixed in next update (obj_info field is skipped), until that could manually remove it from the file with text editor.

**Comment by unitycoder** [comment]
happening again, header changed?

```
 ply
format ascii 1.0
comment Created by CloudCompare v2.11 alpha (Anoia)
comment Created 13/10/2019 18:55
obj_info Generated by CloudCompare!
element vertex 2549173
property float x
property float y
property float z
property uchar red
property uchar green
property uchar blue
end_header
```

**Comment by unitycoder** [comment]
similar issue with header double values:

```
ply
format ascii 1.0
comment Created by CloudCompare v2.11 alpha (Anoia)
comment Created 12/05/2019 12:32
obj_info Generated by CloudCompare!
element vertex 6944196
property double x
property double y
property double z
property uchar red
property uchar green
property uchar blue
end_header
```

**Comment by unitycoder** [comment]
fixed in the coming v2.20 update
